invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -85,9 +102,10 @@ export const RecordVersionsList = (props) => {
   }, []);
 
   return loading ? (
-    <PlaceholderLoader />
+    <>{isPreview ? <PreviewMessage /> : <PlaceholderLoader />}</>
```

Comment:
Sure... it will show up when loading is done so it's not a big deal if `PreviewMessage` is absent here (or place it before the conditional so not to have to repeat it).

Reply:
It's needed there because if the call we do just before (check [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/9f78032067aca5a0b3d2f5a8533de88b4a1df5f0/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js#L86-L94)) doesn't retrieve any results it will stay with the loading state for ever. That's the case for the drafts.